federation/client/clientset_generated/federation_internalclientset/typed/core/unversioned/fake/fake_core_client.go

```diff
@@ -20,6 +20,7 @@ import (
 	unversioned "k8s.io/kubernetes/federation/client/clientset_generated/federation_internalclientset/typed/core/unversioned"
 	restclient "k8s.io/kubernetes/pkg/client/restclient"
 	core "k8s.io/kubernetes/pkg/client/testing/core"
+	fake "k8s.io/kubernetes/pkg/client/unversioned/fake"
```

Comment:
Also, if we decided to import this fake, can you move the fake RESTClient to pkg/client/restclient? Clientsets will replace unversioned clients one day, so we should avoid import pkg/client/unversioned.